client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -203,6 +205,11 @@ const CheckoutThankYou = React.createClass( {
 		return page( `/stats/insights/${ this.props.selectedSite.slug }` );
 	},
 
+	isEligibleForLiveChat() {
+		const { planSlug } = this.props;
+		return planSlug === 'jetpack_business' || planSlug === 'jetpack_business_monthly';
```

Comment:
Lets use the plans constants from `lib/plans/constants` rather than the raw plan strings.

There may also be functions there to abstract away the yearly/monthly and remove the `||` condition here, but not a big deal.